server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiUtil.java

```diff
@@ -55,18 +56,35 @@
      * Returns a newly created {@link HttpResponseException} with the specified {@link HttpStatus} and
      * {@code message}.
      */
-    static HttpResponseException newHttpResponseException(HttpStatus status, String message) {
+    public static HttpResponseException newHttpResponseException(HttpStatus status, String message) {
         return HttpResponseException.of(newResponseWithErrorMessage(status, message));
     }
 
+    /**
+     * Returns a newly created {@link HttpResponseException} with the specified {@link HttpStatus} and
+     * {@link JsonNode}.
+     */
+    public static HttpResponseException newHttpResponseException(HttpStatus status, JsonNode node) {
+        return HttpResponseException.of(newResponseWithJson(status, node));
```

Comment:
Perhaps we should provide a way to specify an `HttpStatus` with a response object in Armeria?

Reply:
I was think about adding methods whose signatures are same as `HttpResponse.of(...)`, but I was not sure which methods I needed to add to `HttpResponseException`, because there are a lot. :-)
To handle above case, I need a method like this.
```java
public static HttpResponseException of(HttpStatus status, MediaType mediaType, byte[] content) {
...
}
or 
public static HttpResponseException of(HttpHeaders headers, byte[] content) {
...
}
```

Reply:
Yeah we should add those methods so that a user can send an error response conveniently. However, I guess a user will find it weird to throw an exception when sending a successful response.

Reply:
That's right. Do we need to check the status here?

Reply:
I guess not. You know, some peculiar API uses 200 OK for error responses ;-)

Reply:
Yeah, we talked about it in this morning. lol